Library/ViewModels/SignupViewModel.swift

```diff
@@ -132,6 +138,18 @@ public final class SignupViewModel: SignupViewModelType, SignupViewModelInputs,
     self.postNotification = self.environmentLoggedInProperty.signal
       .mapConst(Notification(name: .ksr_sessionStarted))
 
+    self.notifyDelegateOpenHelpType = self.tappedUrlProperty.signal.skipNil().map { url -> HelpType? in
+      let helpType = HelpType.allCases.filter { helpType in
```

Comment:
Minor optimization but if we change this from `filter` to `first` then the iteration over the cases stops when it finds a match. Then we can also drop the trailing `first`.